### PR TITLE
Feature: configurable custom font

### PR DESCRIPTION
**Pull request by Freaxed**

### CODE REVIEW

File: src/ui/Editor.cpp [hunk 1]
```diff
@@ -1532,6 +1532,10 @@ void
 Editor::_ApplyExtensionSettings()
 {
 	BFont font = be_fixed_font;
+	const char* fontFamily = gCFG["edit_fontfamily"];
```
Comment (jackburton79) [inline src/ui/Editor.cpp:1535]: I'd use BString here to avoid the strlen

**Review by jackburton79 — CHANGES_REQUESTED** [review]
We shouldn't allow any font family, though: we have to filter out the non-fixed fonts using BFont::IsFixed()

**Comment by Freaxed** [comment]
Not 100% sure we should prevent non-fixed font...

**Comment by nexus6-haiku** [comment]
> Not 100% sure we should prevent non-fixed font...

I agree. We may provide a way to identify if a font is fixed, though.
A filter maybe or a label before the font name?

**Comment by jackburton79** [comment]
> Not 100% sure we should prevent non-fixed font...

Ok I guess you're right. I'm just used to fixed fonts in IDEs

**Review by jackburton79 — APPROVED** [review]

**Pull request merged**